test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -1401,3 +1401,73 @@ Describe "UNC paths" -Tags 'CI' {
         }
     }
 }
+
+Describe "Copy-Item over remote sessions" -Tag 'CI' {
+    BeforeAll {
+        $fullPath          = "C:\Users\Public\"
+        $localTestDir      = "LocalTestDir"
+        $remoteTestDir     = "RemoteTestDir"
+        $localTestDirPath  = Join-Path $fullPath $localTestDir
+        $remoteTestDirPath = Join-Path $fullPath $remoteTestDir
+        $fileName          = "TestFile.txt"
+        $fileContent       = "testContent"
+        $localTestFilePath = Join-Path $localTestDirPath $fileName
+        $remoteTestFilePath = Join-Path $remoteTestDirPath $fileName
+
+        New-Item -ItemType Directory -Path $localTestDirPath
+        New-Item -ItemType Directory -Path $remoteTestDirPath
+    }
+
+    AfterAll {
+        Remove-Item -Path $localTestDirPath -Force -Recurse -ErrorAction SilentlyContinue
+        Remove-Item -Path $remoteTestDirPath -Force -Recurse -ErrorAction SilentlyContinue
+    }
+
+    BeforeEach {
+        $session = New-PSSession -ComputerName .
```

Comment:
Use `New-RemoteSession` helper to create PS Sessions, so the get the correct credentials for working in AppVeyor.

https://github.com/PowerShell/PowerShell/blob/master/test/tools/Modules/HelpersRemoting/HelpersRemoting.psm1#L102